get-started/welcome.qmd

```diff
@@ -0,0 +1,108 @@
+---
+title: "Welcome to Shiny"
+editor:
+  markdown:
+    wrap: sentence
+---
+
+Shiny for Python is a powerful and beginner-friendly framework for building interactive web applications and dashboards, all in pure Python.
+Whether you’re a data scientist, analyst, or developer, Shiny makes it easy to create rich, interactive experiences without needing to learn JavaScript or front-end frameworks.
+
+## Batteries included: everything you need to build a dashboard {#batteries-included}
+
+Shiny for Python comes fully equipped with everything you need to build a dashboard right out of the box.
+It includes a rich set of [input and output components](/components) so you can quickly build interactive applications without worrying about external dependencies.
+Layout options let you organize your UI efficiently, while built-in theming (including dark mode) ensures your app looks great with minimal effort.
+You can even use a [brand.yml](https://posit-dev.github.io/brand-yml/) file to apply consistent branding, colors, and logos across your application.
```

Comment:
This would be a good place to have a row of small card links to the inputs, outputs and components pages

![image](https://github.com/user-attachments/assets/a16d0b14-44d7-4330-8886-86cc86dcfbd0)

I like the idea of including theming/brand.yml, but it might not fit here?

Let me know if you need any assistance with the card links.

Reply:
![image](https://github.com/user-attachments/assets/9167a6bf-888b-484a-9869-8b8f989070f0)